### DISCUSSION TITLE
操作系统的任务调度机制（三）调度器策略 · posts/the-scheduler-policy-of-task-scheduling-in-os/

### DISCUSSION THREAD

**Discussion by giscus** [tcitry/tcitry.github.io#138] [Announcements]
# 操作系统的任务调度机制（三）调度器策略

操作系统中的调度器有任务调度、I/O 调度（以后再说）、内存调度（之前提到的换页策略）等多种类型，本文主要关注任务调度。在单核多进程并发的环境里，进程之间并不是“并行”执行的，实际上在任何时刻都只有一个

https://yindongliang.com/posts/the-scheduler-policy-of-task-scheduling-in-os/

**Comment by schecterdamien** [comment]
mark下，以后再看，不错不错
